spec/request/swagger_spec.rb

```diff
@@ -484,6 +484,13 @@
           )
         end
       end
+
+      it 'supports getting rating info' do
+        expect(subject).to validate(:get, '/v0/disability_compensation_form/find_rating_info_pid', 401)
```

Comment:
Not sure you need to test this here -- seems like it should be a different spec if its important

Reply:
I was doing this because it seems to be consistent with the rest of the spec.  I'm happy to remove it if you don't think it belongs here.  I didn't want to *not* add it and then have someone ask why its missing.